samples/authenticate_implicit_with_adc.rb

```diff
@@ -40,7 +40,7 @@ def authenticate_implicit_with_adc project_id:
   buckets.each do |bucket|
     puts bucket.name
   end
-  puts "Listed all storage buckets."
+  puts "Plaintext: Listed all storage buckets."
```

Comment:
Why this change? (Out of curiosity)

Reply:
😅  I actually can't remember now. I think I put it there while experimenting with sample_loader.